---a/net/minecraft/entity/effect/EntityWeatherEffect.java
+++b/net/minecraft/entity/effect/EntityWeatherEffect.java
@@ -3,13 +3,12 @@
 import net.minecraft.entity.Entity;
 import net.minecraft.world.World;
 
-public abstract class EntityWeatherEffect extends Entity
-{
+public abstract class EntityWeatherEffect extends Entity {
+
     private static final String __OBFID = "CL_00001665";
 
-    public EntityWeatherEffect(World worldIn)
-    {
+    public EntityWeatherEffect(World worldIn) {
         super(worldIn);
     }
-}
+}
